contracts/PosOperator.sol

```diff
@@ -97,9 +97,9 @@ contract PosOperator is PoaOperator {
   // solium-enable security/no-tx-origin
 
   function activate(uint256 _slotId) public {
-    require(_slotId < epochLength);
+    require(_slotId < epochLength, "slot not available");
```

Comment:
inconsistent with "slot not open" in `bet` above. Let's use one of these in both places (both msgs are good, up to you which one to use)

Reply:
used available